store/postgres/src/deployment_store.rs

```diff
@@ -1130,9 +1130,19 @@ impl DeploymentStore {
             )?;
 
             Ok(event)
-        })?;
+        });
 
-        Ok(event)
+        match event {
+            Ok(e) => Ok(e),
+            Err(err) => match err {
+                StoreError::DuplicateBlockProcessing(_deployment_hash, block_number) => {
+                    let default_event = StoreEvent::from_mods(&site.deployment, mods);
```

Comment:
what `default_event` will be ?

Reply:
Something?

Tracing the stacks upward we can see that, once this function completes without error, no one gonna give a shit about its result.

Note that `site.deployment` & `mods` are immutable inputs as well - meaning they bear no effects to the state of the application either